pkg/refactor/mirror/parser.go

```diff
@@ -34,18 +36,19 @@ type Parser struct {
 	DeprecatedWorkflowVars  map[string]map[string][]byte
 }
 
-func NewParser(log FormatLogger, src Source) (*Parser, error) {
+func NewParser(namespace string, src Source) (*Parser, error) {
```

Comment:
This function had a number of errors & warnings that were intended to show up in the mirror activity logs. If I understand correctly, without passing in a logger here -- or at least a process id -- I think those logs will fail to show up. They'll be printed to stdout and probably show up in namespace logs, but not process logs. We should change this.

Reply:
We need to merge now due to many changes on main, but we can talk about the mirror logs next week